components/elements/buttons/BaseButton/BaseButton.vue

```diff
@@ -1,3 +1,7 @@
+<script setup lang="ts">
+// blargh
```

Comment:
do we want this to say something other than blargh?

Reply:
addressed in discord.